test/spec/xcSessionSpec.ts

```diff
@@ -37,13 +42,14 @@ describe("Test xcSession module", function () {
         let session;
         beforeEach(function () {
             let serverUrl = "wss:\\serverUrl";
-            let mockWebSocket = new WebSocket(serverUrl);
+            window["WebSocket"] = WebSocket;
```

Comment:
@nicolaserny  is doing this differently in the other PR...

https://github.com/xcomponent/ReactiveXComponent.js/pull/31/files#diff-a9ae241995226095ebf400b83a19e1bdR16

we'd better synchronize sooner than later.